gensrc/script/functions.py

```diff
@@ -710,6 +710,11 @@
     [120151, "md5sum_numeric", "LARGEINT", ["VARCHAR", "..."], "EncryptionFunctions::md5sum_numeric", False],
     [120160, "sha2", "VARCHAR", ["VARCHAR", "INT"], "EncryptionFunctions::sha2", "EncryptionFunctions::sha2_prepare", "EncryptionFunctions::sha2_close", False],
     [120161, "to_base64", "VARCHAR", ["VARBINARY"], "EncryptionFunctions::to_base64", False],
+    [120170, "fpe_encrypt", "VARCHAR", ["VARCHAR", "VARCHAR"], "EncryptionFunctions::fpe_encrypt_num", False],
+    [120171, "fpe_encrypt_num", "VARCHAR", ["VARCHAR", "VARCHAR"], "EncryptionFunctions::fpe_encrypt_num", False],
```

Comment:
Is fpe_encrypt_num alias of fpe_encrypt? if so, please use fe/fe-core/src/main/java/com/starrocks/sql/parser/SyntaxSugars.java to translate the alias into canonical name.

Reply:
I Fixed It, These are 4 different functions